docs/developer-docs/web-apps/verifiable-credentials/issuer.mdx

```diff
@@ -0,0 +1,184 @@
+---
+keywords: [intermediate, tutorial, identity, credentials, verifiable credentials, issuer]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Become an issuer
+
+<MarkdownChipRow labels={["Intermediate", "Tutorial"]} />
+
+## Overview
+
+An **issuer** is a service or app that can issue a verifiable credential (VC) to a user. On ICP, an issuer is an exposed API that recieves calls from an identity provider and does not trigger any workflows itself. To become an issuer, your canister must implement the issuer API as described in [verifiable credential spec](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).
```

Comment:
I think the original order works better, since it describes how an issuer on ICP specifically is implemented (exposed API) then provides the instructions on how to implement it.